templates/rpm/project.spec.erb

```diff
@@ -125,12 +130,30 @@ done
 <%= @platform.add_user(@user) %>
 <%- end -%>
 
+# Run preinstall scripts on install if defined
+if [ $1 -eq 1 ] ; then
+  <%= get_preinstall_actions("install") %>
+fi
+
+# Run preinstall scripts on upgrade if defined
+if [ $1 -eq 2 ] ; then
+  <%= get_preinstall_actions("upgrade") %>
+fi
+
+
+
 %post
 <%- if @platform.is_aix? || (@platform.is_el? && @platform.os_version.to_i == 4) -%>
   ## EL-4 and AIX RPM don't have %posttrans, so we'll put them here
-  <%- get_postinstall_actions.each do |task| -%>
-    <%= task %>
-  <%- end -%>
+  # Run postinstall scripts on install if defined
```

Comment:
Indentation here is inconsistent with the earlier scripts. Same issue 5 lines down.